source/docs/software/vision-processing/apriltag/apriltag-intro.rst

```diff
@@ -0,0 +1,184 @@
+.. include:: <isonum.txt>
+
+What Are AprilTags?
+===================
+
+.. image:: images/apriltagrobots_overlay.png
+
+AprilTags are a system of visual tags developed by researchers at the University of Michigan to provide low overhead, high accuracy localization for many different applications.
+
+Application to FRC
+------------------
+
+In the context of FRC, AprilTags are useful for helping your robot know where it is at on the field, so it can align itself to some goal position.
+
+AprilTags have been in development since 2011, and have been refined over the years to increase the robustness and speed of detection.
+
+Starting in 2023, FIRST is providing a number of tags, scattered throughout the field, each at a known :term:`pose`.
+
+All of the tags are from the 36H11 family, which means that:
+
+1) Each tag carries 36 bits of information
+2) It would take at least 11 bit "flips" before one tag could be mistaken for another.
+
+All tags will be printed such that their outer black border is 200mm on a side.
```

Comment:
From the [FRC Blog](https://www.firstinspires.org/robotics/frc/blog/2022-control-system-reporting-2023-updates-and-beta-testing):

>The tags will be approximately an 8 ⅛ in. square. This means that the black square is 6.5 in. from outside to outside and should be printable by teams on any standard printer.

The 200mm number was what PhotonVision was using for the offseason tests before the announcement from HQ.

Likewise the PDFs on L26 may be 200mm and undersized in relation to the official field tags, but more testing is required.